mips2c_macros.h

```diff
@@ -33,4 +33,8 @@ typedef s64 MIPS2C_UNK64;
 #define MIPS2C_BREAK() (0)
 #define MIPS2C_SYNC() (0)
 
+/* Floating-point intrinisics */
+#define MIPS2C_FRES(x) (1.0 / x)            /* PPC reciprocal estimator (fres) */
+#define MIPS2C_FRSQRTE(x) (1.0 / sqrtf(x)   /* PPC inverse square root estimator (frsqrte) */
```

Comment:
I haven't been able to get these instructions to show up without using the built-in intrinsic (`__frsqrte(x)`/`__fres(x)`).
They both seem ~somewhat common, they show up in more functions than the PS instructions, but this could be from a macro or inlining. 

I could also just emit `__frsqrte(...)`/`__fres(...)` instead of the `MIPS2C_FRSQRTE` bit? These both have helpful top results on google, and would compile (with MWCC at least...)

Reply:
That sounds like a good idea!